composer.json

```diff
@@ -1,48 +1,58 @@
 {
   "name": "e0ipso/travis-drupal-module",
-  "type": "composer-plugin",
   "description": "Library to help Travis CI setup without headaches for Drupal modules.",
+  "license": "GPL-2.0",
+  "type": "composer-plugin",
   "keywords": [
     "Drupal",
     "Travis"
   ],
-  "homepage": "https://github.com/e0ipso/travis-drupal-module",
-  "license": "GPL-2.0",
   "authors": [
     {
       "name": "Mateu Aguiló Bosch",
       "email": "mateu@mateuaguilo.com"
     }
   ],
+  "homepage": "https://github.com/e0ipso/travis-drupal-module",
+  "support": {
+    "issues": "https://github.com/e0ipso/travis-drupal-module/issues",
+    "source": "https://github.com/e0ipso/travis-drupal-module"
+  },
   "require": {
     "php": ">=7.3",
-    "composer-plugin-api": "^1.1",
-    "dealerdirect/phpcodesniffer-composer-installer": "^0.5.0",
+    "composer-plugin-api": "^1.1 || ^2.0",
+    "dealerdirect/phpcodesniffer-composer-installer": "^0.5.0 || ^1.0",
     "drupal/coder": "^8.2",
     "jakub-onderka/php-parallel-lint": "^1.0",
-    "localheinz/composer-normalize": "^1.3",
+    "localheinz/composer-normalize": "^2.4",
     "nikic/php-parser": "^4.2",
     "phpmd/phpmd": "^2",
     "phpmetrics/phpmetrics": "^2.4",
-    "phpro/grumphp": "^0.15.2",
```

Comment:
We should look into using the phar instead.

https://github.com/phpro/grumphp#installation

Reply:
Done!